packages/wonder-blocks-form/src/components/__docs__/checkbox.stories.js

```diff
@@ -0,0 +1,156 @@
+// @flow
+import * as React from "react";
+import {StyleSheet} from "aphrodite";
+
+import {View} from "@khanacademy/wonder-blocks-core";
+import {LabelMedium, LabelSmall} from "@khanacademy/wonder-blocks-typography";
+import type {StoryComponentType} from "@storybook/react";
+
+import Checkbox from "../checkbox.js";
+
+import ComponentInfo from "../../../../../.storybook/components/component-info.js";
+import {name, version} from "../../../package.json";
+
+export default {
+    title: "Form / Checkbox",
+    component: Checkbox,
+    parameters: {
+        componentSubtitle: ((
+            <ComponentInfo name={name} version={version} />
+        ): any),
+    },
+};
+
+export const CheckboxDefault: StoryComponentType = (args) => (
+    <Checkbox {...args} />
+);
+
+CheckboxDefault.args = {
+    checked: false,
+    onChange: () => {},
+};
+
+export const CheckboxBasic: StoryComponentType = () => {
+    const [checked, setChecked] = React.useState(false);
+    return <Checkbox checked={checked} onChange={setChecked} />;
+};
+
+CheckboxBasic.parameters = {
+    docs: {
+        storyDescription:
+            "Use state to keep track of whether the checkbox is checked or not",
+    },
+};
+
+export const CheckboxVariations: StoryComponentType = () => (
```

Comment:
Nit: oh maybe we could use the term `Variants` here. This is a popular term in Design Systems and Figma as well, so we can have some consistency between Design and Eng :) (cc @megaroeny in case you have any thoughts).

https://www.figma.com/best-practices/creating-and-organizing-variants/